### DISCUSSION TITLE
Chrome top is missing‏

### DISCUSSION THREAD

**Discussion by Ahron333** [cefsharp/CefSharp#4738] [Q&A]
Hey i am trying to display my chromium buttons not only the webview, The Refreash button back & forth search bar and all of the top button/options on chrome example:
![Screenshot_51](https://github.com/cefsharp/CefSharp/assets/37657144/59386206-eecb-4776-8845-6b810fd7b99c)

The result that i am getting when loading a webpage on chromium:
![image](https://github.com/cefsharp/CefSharp/assets/37657144/cc06be39-12d0-4184-9842-de962cd4e3d3)

As you can see the top is not there. I know that you can make custom buttons like back and forth and the searchabar But can i some display the original chrome top and have that displayed on a panel like the webview?

**Comment by amaitland** [comment]
> i am trying to display my chromium buttons

Not supported.

> But can i some display the original chrome top and have that displayed on a panel like the webview?

No

You'd need to implement the functionality yourself or look for a different library.

**Reply by Ahron333** [reply]
Does that was supported?

**Reply by amaitland** [reply]
I don't understand the question. What do you mean?
